### PR TITLE
Chapter 8

### PR DESCRIPTION
**Pull request by tuongvule**
Screenshot from 2021-01-28 04-38-26

### CODE REVIEW

**Comment by tuongvule** [comment]
image

**Comment by tuongvule** [comment]
ready

File: app/assets/stylesheets/sessions.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the Sessions controller here.
```
Comment (vanvtt-0952) [inline app/assets/stylesheets/sessions.scss]: Xóa file

File: app/views/sessions/create.html.erb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+<h1>Sessions#create</h1>
```
Comment (vanvtt-0952) [inline app/views/sessions/create.html.erb]: I18n

File: app/views/sessions/destroy.html.erb [hunk 3]
```diff
@@ -0,0 +1,2 @@
+<h1>Sessions#destroy</h1>
```
Comment (vanvtt-0952) [inline app/views/sessions/destroy.html.erb]: I18n

File: app/views/sessions/new.html.erb [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+  <div class="col-md-6 col-md-offset-3">
+    <%= form_for :session, url: login_path do |f| %>
+      <p style="font-weight: bold"><%= t(".email")%></p>
+      <%= f.email_field :email, class: "form-control" %>
+      <p style="font-weight: bold"><%= t(".password")%></p>
+      <%= f.password_field :password, class: "form-control" %>
+      <%= f.submit t(".login"), class: "btn btn-primary" %>
+    <% end %>
+    <p><en><%= t (".new_user") %></en> <en><%= link_to t(".sign_up_now"),
+                                       signup_path %></en></p>
```
Comment (vanvtt-0952) [inline app/views/sessions/new.html.erb]: indent

File: test/controllers/users_controller_test.rb [hunk 5]
```diff
@@ -0,0 +1,7 @@
+require 'test_helper'
+
```
Comment (vanvtt-0952) [inline test/controllers/users_controller_test.rb]: Xóa file

**Comment by tuongvule** [comment]
ready

File: test/controllers/users_controller_test.rb (hunk 5 above)
Comment (tuongvule) [reply test/controllers/users_controller_test.rb]: đã xong chị Vân nhé

File: app/views/sessions/new.html.erb [hunk 6]
```diff
@@ -0,0 +1,15 @@
+<% provide :title, t(".login_title")%>
+<h1 class="center"><%= t(".login") %></h1>
+<div class="row">
+  <div class="col-md-6 col-md-offset-3">
+    <%= form_for :session, url: login_path do |f| %>
+    <p style="font-weight: bold"><%= t(".email")%></p>
```
Comment (vanvtt-0952) [inline app/views/sessions/new.html.erb]: indent

File: app/views/sessions/new.html.erb [hunk 7]
```diff
@@ -0,0 +1,15 @@
...
+    <%= f.email_field :email, class: "form-control" %>
+    <p style="font-weight: bold"><%= t(".password")%></p>
+    <%= f.password_field :password, class: "form-control" %>
+    <%= f.submit t(".login"), class: "btn btn-primary" %>
+    <% end %>
+    <p><en><%= t (".new_user") %></en> <en><%= link_to t(".sign_up_now"),
+    signup_path %></en></p>
```
Comment (vanvtt-0952) [inline app/views/sessions/new.html.erb]: indent

File: app/views/static_pages/home.html.erb [hunk 8]
```diff
@@ -1,9 +1,11 @@
 <% provide :title, t("pages.home.home_title") %>
 <div class="center jumbotron">
-  <h1><%= t "pages.home.welcome_to_the_sample_app"%></h1>
+  <h1><%= t "pages.home.welcome_to_the_sample_app" %></h1>
   <h2>
-    <%= t("pages.home.this_is_the_home_page", link: link_to(t("pages.home.ruby_on_rails_tutorial"),
+    <%= t("pages.home.this_is_the_home_page",
+    link: link_to(t("pages.home.ruby_on_rails_tutorial"),
```
Comment (vanvtt-0952) [inline app/views/static_pages/home.html.erb]: indnet

**Comment by vanvtt-0952** [comment]
merged
2 comment về indent, fix đi rồi mợt

**Pull request merged**